Mathlib/GroupTheory/Subgroup/Centralizer.lean

```diff
@@ -94,4 +94,22 @@ abbrev closureCommGroupOfComm {k : Set G} (hcomm : ∀ x ∈ k, ∀ y ∈ k, x *
       have := closure_le_centralizer_centralizer k
       Subtype.ext <| Set.centralizer_centralizer_comm_of_comm hcomm _ (this h₁) _ (this h₂) }
 
+/-- The homomorphism N(H) → Aut(H) with kernel C(H). -/
+@[simps]
+def normalizerMonoidHom : H.normalizer →* MulAut H where
+  toFun := fun g ↦
+    { toFun := fun h ↦ ⟨g * h * g⁻¹, (g.2 h).mp h.2⟩
+      invFun := fun h ↦ ⟨g⁻¹ * h * g, (mem_normalizer_iff''.mp g.2 h).mp h.2⟩
+      left_inv := fun _ ↦ by simp [mul_assoc]
+      right_inv := fun _ ↦ by simp [mul_assoc]
+      map_mul' := by simp [mul_assoc] }
+  map_one' := by simp [DFunLike.ext_iff]
+  map_mul' := by simp [DFunLike.ext_iff, mul_assoc]
+
+/-- The N/C theorem: N(H)/C(H) is isomorphic to a subgroup of Aut(H). -/
```

Comment:
This theorem says nothing about the quotient, only about the kernel. If you want to claim `N/C`, then you should add a definition for an injective hom `N/C ->* MulAut H` by using the quotient group API.

Reply:
Fair enough. I'll change the docstring. I don't think having the quotient map is super helpful.